app/app.rb

```diff
@@ -1,21 +1,25 @@
 require_relative '../classes/genre'
 require_relative '../classes/music_album'
+require_relative '../classes/game'
+require_relative '../classes/author'
 require_relative '../classes/book'
 require_relative '../classes/label'
 
 class App
-  attr_reader :genres, :albums, :books, :labels
+  attr_reader :genres, :albums, :books, :labels, :games, :authors
 
   def initialize(params = { data_dir: 'data' })
     @data_dir = params[:data_dir]
     @genres = params[:genres] || load(:genres)
     @albums = params[:albums] || load(:albums)
+    @games = params[:games] || load(:games)
+    @authors = params[:authors] || load(:authors)
     @labels = params[:labels] || load(:labels)
     @books = params[:books] || load(:books)
   end
 
   def load(collection)
-    [] if %i[genres albums labels books].include?(collection)
+    [] if %i[genres albums labels books games authors].include?(collection)
```

Comment:
## 👏 well done!!
Reusing what we had and avoid app crashes

Reply:
@luigirazum I used what we had, I only just added my part of the task, which are games and authors.
here:
[] if %i[genres albums labels books games authors].include?(collection)

except I don't understand what you are asking then you can please provide suggestions.

Thanks